src/config.ts

```diff
@@ -170,7 +169,10 @@ export default {
       ? true
       : ENABLE_COMMERCE_STITCHING === "true",
   ENABLE_HEAPDUMPS,
-  FORCE_URL,
+  FORCE_URL:
+    FORCE_URL || NODE_ENV === "production"
+      ? "https://artsy.net"
+      : "https://staging.artsy.net",
```

Comment:
I'm not sure `NODE_ENV` is a good thing to switch off of. Even in staging, we run `NODE_ENV` as production. We could just default it to `https://www.artsy.net` if it's undefined, otherwise leave it out and people can figure out what they need if/when it errors.